tools/gfx/d3d12/render-d3d12.cpp

```diff
@@ -5407,8 +5422,8 @@ Result D3D12Device::createComputePipelineState(const ComputePipelineStateDesc& i
         D3D12_COMPUTE_PIPELINE_STATE_DESC computeDesc = {};
         computeDesc.pRootSignature = programImpl->m_rootObjectLayout->m_rootSignature;
         computeDesc.CS = {
-            programImpl->m_computeShader.getBuffer(),
```

Comment:
Should there be a check that there is a m_shaders.getCount() == 1 and it's the right type?

Reply:
All these checks are supposed to be done in debug layer, and are omitted here to avoid overhead.